content/source/docs/enterprise/api/index.html.md

```diff
@@ -107,3 +107,56 @@ Although most of our API endpoints use the POST method and receive their paramet
 Since these parameters were originally designed as part of a JSON object, they sometimes have characters that must be [percent-encoded](https://en.wikipedia.org/wiki/Percent-encoding) in a query parameter. For example, `[` becomes `%5B` and `]` becomes `%5D`.
 
 For more about URI structure and query strings, see [the specification (RFC 3986)](https://tools.ietf.org/html/rfc3986) or [the Wikipedia page on URIs](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier).
+
+### Inclusion of Related Resources
+
+Some of our GET endpoints have the ability to return additional information about nested resources by adding an `?include` parameter to the query.
```

Comment:
I'd probably prefer "Some GET endpoints", or "Some of the API's GET endpoints". (Or maybe even "Some of the API's List and Show endpoints"?)